app/code/Magento/CustomerGraphQl/composer.json

```diff
@@ -4,15 +4,15 @@
   "type": "magento2-module",
   "require": {
     "php": "~7.1.3||~7.2.0||~7.3.0",
-    "magento/module-customer": "*",
```

Comment:
I deleted because is [duplicated](https://github.com/magento/graphql-ce/pull/1031/files#diff-a8af043b769ff64a7015b022a14e658dR8)